src/components/contextual/pages/vebal/LMVoting/GaugesTableMyVotes.vue

```diff
@@ -6,12 +6,9 @@ import BigNumber from 'bignumber.js';
 import useNumbers from '@/composables/useNumbers';
 import useVotingEscrowLocks from '@/composables/useVotingEscrowLocks';
 import { useI18n } from 'vue-i18n';
-import {
-  isVotingTimeLocked,
-  remainingVoteLockTime,
-} from '@/composables/useVeBAL';
-import TimelockIcon from '@/components/_global/icons/TimelockIcon.vue';
+import { isVotingTimeLocked } from '@/composables/useVeBAL';
 import BalTooltip from '@/components/_global/BalTooltip/BalTooltip.vue';
+import TimeLockedVote from '../MultiVoting/TimeLockedVote.vue';
```

Comment:
Ideally, this shared component would go in a shared module but I preferred to keep it inside MultiVoting path as it is the only exception.